### PR TITLE
Feature/px web2 231 add alert to variablebox

### PR DESCRIPTION
**Pull request by SjurSutterudSagen**
Since we want to show when there are no selected values for a mandatory
variable, we need to add this capability to the VariableBox component.

When no value is selected, it now shows and alert in the header of the
component, and the border is colored red. This is removed when a value
is chosen.

API handling is not added, that would be a different task.

Also had to add a new folder to ignore files. This is for the newest changes 
in Nx, and those lines are also added in the PR for those updates.

### CODE REVIEW

**Comment by SjurSutterudSagen** [comment]
Checking to see if the animations in the design will take alot of time or not. If not I'll add them too.

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBox.tsx [hunk 1]
```diff
@@ -52,6 +52,9 @@ export function VariableBox({
   const chosenValuesLength = selectedValues.find(
     (variables) => variables.id === id
   )?.values.length;
+  const currentVariable = selectedValues.find((variable) => variable.id === id);
+  const hasMandatoryNoSelectedValuesError =
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/VariableBox/VariableBox.tsx:56]: Consider using another variable name. This one is a bit difficult to follow. Maybe isMissingMandatoryValueError?

**Review by michaelpande — APPROVED** [review]
Nothing big, just some minor comments. Looks good 👍

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxHeader/VariableBoxHeader.tsx [hunk 2]
```diff
@@ -52,40 +55,56 @@ export function VariableBoxHeader({
...
         </div>
       </div>
 
-      <div className={cssClasses}>
-        {isOpen ? (
-          <Icon iconName="ChevronUp"></Icon>
-        ) : (
-          <Icon iconName="ChevronDown"></Icon>
+      <div className={cl(classes['header-alert'])}>
+        {isMissingMandatoryValues && (
```
[the remark this answers is not in the document]
Comment (SjurSutterudSagen) [reply libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxHeader/VariableBoxHeader.tsx:102]: Yes, I'll fix it :)

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBox.tsx (hunk 1 above)
Comment (SjurSutterudSagen) [reply libs/pxweb2-ui/src/lib/components/VariableBox/VariableBox.tsx]: Changed to use this name

**Review by PerIngeVaaje — APPROVED** [review]
OK

**Pull request merged**